pom.xml

```diff
@@ -0,0 +1,168 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.github.giis-uniovi</groupId>
+    <artifactId>retorch</artifactId>
+    <version>1.1.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>retorch</name>
+    <description>RETORCH: Resource-aware Orchestration of End-to-End Test Cases
+    </description>
+    <url>https://github.com/giis-uniovi/retorch</url>
+    <organization>
+        <name>Software Engineering Research Group (GIIS) - University of Oviedo, ES</name>
+        <url>http://giis.uniovi.es/</url>
+    </organization>
+
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <modules>
+        <module>retorch-llm-rp</module>
+    </modules>
```

Comment:
Do you plan to use THIS repo to other works, if so, it is ok to keep it as multimodule, if not, maybe this is not necessary

Reply:
@javiertuya  I was thinking about rename the current project to retorch-llm-expstudy and leave the name of the repository retorch-llm-rp. What do you think?